ndash/src/chunk/container_media_chunk.cc

```diff
@@ -198,12 +200,12 @@ void ContainerMediaChunk::GiveSeekMap(
   // Do nothing.
 }
 
-std::unique_ptr<MediaFormat> ContainerMediaChunk::GetAdjustedMediaFormat(
-    const MediaFormat* format,
+std::unique_ptr<const MediaFormat> ContainerMediaChunk::GetAdjustedMediaFormat(
+    std::unique_ptr<const MediaFormat> format,
     base::TimeDelta sample_offset) {
-  std::unique_ptr<MediaFormat> out;
+  std::unique_ptr<const MediaFormat> out;
 
-  if (format == nullptr) {
+  if (format.get() == nullptr) {
```

Comment:
nit: You should be able to just do if (!format) ...

Reply:
Done